pytorch_lightning/core/grads.py

```diff
@@ -36,20 +38,12 @@ def grad_norm(self, norm_type: Union[float, int, str]) -> Dict[str, float]:
             norms: The dictionary of p-norms of each parameter's gradient and
                 a special entry for the total p-norm of the gradients viewed
                 as a single vector.
-        """
-        norm_type = float(norm_type)
-
-        norms, all_norms = {}, []
-        for name, p in self.named_parameters():
-            if p.grad is None:
-                continue
-
-            param_norm = float(p.grad.data.norm(norm_type))
-            norms[f'grad_{norm_type}_norm_{name}'] = round(param_norm, 4)
 
-            all_norms.append(param_norm)
-
-        total_norm = float(torch.tensor(all_norms).norm(norm_type))
-        norms[f'grad_{norm_type}_norm_total'] = round(total_norm, 4)
-
-        return norms
+        .. deprecated::v1.3
```

Comment:
```suggestion
        .. deprecated:: v1.3
```
I think without the space sphinx doesn't render it properly